Internationalize content (#4)
ambanum/ParisCall: README.md

# ParisCall
Public website for the Paris Call for Trust and Security in Cyberspace / Site public de l'Appel de Paris pour la confiance et la sécurité dans le cyberespace

## How to add a language

Add the language in `_config.yml`:

```
available_languages: [ en, fr ]
```

Add translations for the language in the `data/translations` files:

```
lang:
  en: English
  fr: Français
menu:
  home:
    en: Home
    fr: Accueil
```

Duplicate `_pages/en` content in `_pages/$language_code` and translate its content.

> Change all `permalink` attributes in front matters.

Reviewer: Aren't we missing instructions on `_supporters_$lang`?

Author: We are, I going to add this.

# ParisCall
Public website for the Paris Call for Trust and Security in Cyberspace / Site public de l'Appel de Paris pour la confiance et la sécurité dans le cyberespace

## How to add a language

Add the language in `_config.yml`:

```

> available_languages: [ "en", "fr" ]

Reviewer: > Not sure quotes are needed in YAML arrays.

```

Add translations for the language in the `data/translations.yml`:

```
lang:
  en: "English"
  fr: "Français"
  es: "Español"
menu:
  home:
    en: "Home"
    fr: "Accueil"
    es: "Inicio"
```

> Duplicate `_pages/en` content in `_pages/es` and translate its content.

Reviewer: ```suggestion
Duplicate `_pages/en` content in `_pages/$language_code` and translate its content.
```

> Its important to change all `lang` and `permalink` in all front matter.

Reviewer: ```suggestion
Change all `lang` and `permalink`  attributes in front matters.
```

> Its important to change all `lang` and `permalink` in all front matter.

Reviewer: Why don't we rely on `_config` defaults to set the `lang` instead of individual front-matters?

Author: No specific reason, I will modify this
